### PR TITLE
feat: Add Configuring Workload identity federation between GCP and AWS EKS

### PR DESCRIPTION
**Pull request by spangenberg**

### CODE REVIEW

File: blog/configuring-workload-identity-federation-between-gcp-and-aws-eks.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: Configuring Workload identity federation between GCP and AWS EKS
+tags:
+  - security
+date: '2022-08-04T00:00:00'
+description: Configure Workload identity federation between GCP and AWS EKS
+authors: danielspangenberg
```
Comment (dj-stormtrooper) [inline blog/configuring-workload-identity-federation-between-gcp-and-aws-eks.md:7]:
```suggestion
authors: danielspangenberg
image: "https://docs.cloudquery.io/og-image/Configuring Workload identity federation between GCP and AWS EKS"
```

It's for meta image (thumbnale twitter, linkedin and etc), autogenerated by Abyssale SaaS

Pattern is `https://docs.cloudquery.io/og-image/<your image text here>`, you can change text if you need

Comment (spangenberg) [reply blog/configuring-workload-identity-federation-between-gcp-and-aws-eks.md:7]: thx

File: blog/configuring-workload-identity-federation-between-gcp-and-aws-eks.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+image: /img/blog_covers/configuring-workload-identity-federation-between-gcp-and-aws-eks.jpg
```
[the remark this answers is not in the document]
Comment (spangenberg) [reply blog/configuring-workload-identity-federation-between-gcp-and-aws-eks.md:8]: Ah okay now I get it... didn't feel right to me :)

[the remark this answers is not in the document]
Comment (dj-stormtrooper) [reply blog/configuring-workload-identity-federation-between-gcp-and-aws-eks.md:8]: Yeah, dark magic a little bit

**Comment by dj-stormtrooper** [comment]
Nice! 
image

**Review by dj-stormtrooper — APPROVED** [review]

**Pull request merged**